### DISCUSSION TITLE
Is there a method for clear text in a text field?

### DISCUSSION THREAD

**Discussion by proximilian** [leancodepl/patrol#1280] [Q&A] [answered]
Hi guys,
One very quick question:
In order to clear previously entered text in a text field I may use $(Key).enterText('') as a workaround, but do we have a direct predefined method for that?

**Comment by bartekpacia** [comment, marked as answer]
Hi @proximilian,

No, we don't have that feature. You can create your own extension method though :)

**Reply by proximilian** [reply]
Ok. Thanks for the quick response. I will. :)
